homeassistant/components/vesync/number.py

```diff
@@ -28,22 +28,24 @@
 class VeSyncNumberEntityDescription(NumberEntityDescription):
     """Class to describe a Vesync number entity."""
 
-    exists_fn: Callable[[VeSyncBaseDevice], bool]
+    exists_fn: Callable[[VeSyncBaseDevice], bool] = lambda _: True
     value_fn: Callable[[VeSyncBaseDevice], float]
-    set_value_fn: Callable[[VeSyncBaseDevice, float], bool]
+    native_min_value_lambda: Callable[[VeSyncBaseDevice], float]
+    native_max_value_lambda: Callable[[VeSyncBaseDevice], float]
```

Comment:
we use `_fn` as suffix for all functions, so let's stay consistent :)

Reply:
Corrected.